src/FieldElementSubsystems.Test/Point/PointTest.cs

```diff
@@ -63,6 +73,52 @@ public void Test_Default_Position()
         Assert.Equal(GenericPointPosition.NoEndPosition, point.PointState.PointPosition);
     }
 
+    [Fact]
+    public async Task Test_Send_Generic_Message()

```

Comment:
Generic message has a meaning with regards to sci already (e.g. initialization start / end, pdi not available are generic sci messages). Naming suggesion: leave out 'Generic', i.e. `SendSciMessage`